devicon.json

```
@@ -6781,6 +6781,41 @@
             }
         ]
     },
+    {
+        "name": "leetcode",
+        "altnames": [],
+        "tags": [
+            "online-platform",
+            "coding-platform",
+            "platform",
+            "coding",
+            "dsa",
+            "interview-preparation"
+        ],
+        "versions": {
+            "svg": [
+                "original",
+                "plain",
+                "line"
+            ],
+            "font": []
```

The font versions need to be added
```suggestion
            "font": [
                "plain",
                "line"
            ]
```